src/driver/driver_api.cc

```diff
@@ -442,68 +442,6 @@ std::pair<IRModule, IRModule> SplitMixedModule(IRModule mod_mixed, const Target&
   return {host_mod, device_mod};
 }
 
-runtime::Module PreProcessModuleForBuild(const Map<Target, IRModule>& inputs_arg,
```

Comment:
I would almost prefer we use this name or maybe come up with an even better one from build especially in internal APIs since it is not very clear what `build` means. We have to preserve for historical API compatibility for now but seems better to not keep in C++ if we can.

Reply:
Good point, since `build` is used pretty frequently elsewhere, and could be confusing to users.  Since most places use "build" to refer to generating something that is executable, I thought it would be appropriate here.  But I think I agree, since in most places "build" has an entire sequence of steps, and here there's only the one.

I'm not a big fan of the name `PreProcessModuleForBuild`, because I read "PreProcess" as implying that this is a pretty early step in the build pipeline, rather than being nearly the last step.

The best name coming to mind is `CodeGenerationForLowLevelTIR` or `GenerateCodeForLowLevelTIR`, but waiting for the caffeine to hit may result in better names.

Reply:
Looking at it with fresh post-weekend eyes, it looks like this function is already part of the external API, [as one of the overloads for `tvm::build`](https://github.com/apache/tvm/blob/main/include/tvm/driver/driver_api.h#L156).

As a proposal for naming conventions, I've made an internal `tvm::TIRToRuntime`, with the name chosen to mimic the [existing target-specific codegen hook](https://github.com/apache/tvm/blob/main/src/target/codegen.cc#L50).  All internal usage calls `tvm::TIRToRuntime` directly, while the `tvm::build` overload is maintained for the external API.